Dockerfile.ol9.mysql-8.0

```diff
@@ -0,0 +1,41 @@
+FROM oraclelinux:9
+LABEL org.opencontainers.image.authors="sjmudd@pobox.com"
+LABEL description="MySQL DEV environment (OL9)"
+
```

Comment:
Since you're setting the author to your name rather than this repository. I suggest to add `org.opencontainers.image.licenses` with GPL or similar license :). Thoughts?

Reply:
I've removed my name to keep things simple.